code/modules/organs/organ_external.dm

```diff
@@ -214,6 +224,11 @@
 					droplimb(1)
 					return
 
+				//burn
+				if(burn > 30 && prob(3 * brute))
```

Comment:
Why is is checking for brute and burn if its just burn related?

Reply:
oh shit, good catch